<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Discord Custom RP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #F2F3F5;
font-weight: 400;
background-color: #242428;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_appid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <property name="text">
     <string>awdswadsdwas</string>
    </property>
    <property name="maxLength">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_appid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Application ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="select_presets">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <item>
     <property name="text">
      <string>Default</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test 2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_presets">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Preset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_details">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_details">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <property name="text">
     <string>My very nice details!</string>
    </property>
    <property name="maxLength">
     <number>400</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_state">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <property name="text">
     <string>A very nice custom State!</string>
    </property>
    <property name="maxLength">
     <number>400</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb_party">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Party</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="number_partyMin">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>181</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="number_partyMax">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="divider">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_timestamp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Timestamp</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="timestampBtns">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
margin-top: 2px;
border: none;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radio_none">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="text">
      <string>None</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_localtime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="text">
      <string>My local time</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_appstart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="text">
      <string>Since app started</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_custom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="text">
      <string>Custom timestamp</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #111214;
border-radius: 7px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_save">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>62</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #4E5058;
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="date_customtimestamp">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #4E5058;
border: none;
border-radius: 7px;
padding: 0 7px;
margin-top: 2px;</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>input_appid</zorder>
   <zorder>input_details</zorder>
   <zorder>input_state</zorder>
   <zorder>lb_appid</zorder>
   <zorder>lb_details</zorder>
   <zorder>lb_party</zorder>
   <zorder>lb_presets</zorder>
   <zorder>lb_state</zorder>
   <zorder>lb_timestamp</zorder>
   <zorder>number_partyMax</zorder>
   <zorder>number_partyMin</zorder>
   <zorder>select_presets</zorder>
   <zorder>timestampBtns</zorder>
   <zorder>divider</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
